### DISCUSSION TITLE
Face replace

### DISCUSSION THREAD

**Discussion by realJogicodes** [lllyasviel/Fooocus#1082] [Ideas]
A multi image prompt could be useful where a face is dropped into a different scene. Kinda how the face swap works already but not onto a text prompt, rather onto a second image prompt. (Maybe that's a thing already and I just am not getting how)

**Comment by carpediembr** [comment]
Is there a way to use the FaceSwap prompt, but on a different pose, as in ... a person looking up or to the side? 

All I'm able to do is copy the front-facing look.

**Reply by rotilho** [reply]
Did you find how to do that?

**Reply by tbuyle** [reply]
I had some success using multiple references images of the person.

Here, I generated in a single image a front and face picture (the two small one on grey background). Then  use them both as faceswap image prompt. Giving more weight to one

 or the other changes the angle in the generated picture : Top image use the default 0,9 default wight for both, the middle has 0,9 front and 0,3 side and the last one 0,7 front and 0,9 side.

![Design sans titre](https://github.com/lllyasviel/Fooocus/assets/467350/d6f4c277-0880-462f-80eb-e342088e5b36)

**Comment by realJogicodes** [comment]
No clue, sorry but you've peaked my attention. How are you doing that?

I tried the image on image prompt but that didn't quite work out for me

**Comment by Phrixus2023** [comment]
here https://www.youtube.com/watch?v=wOBcNPFuB70
